documentation/auto-discovery/topics/advanced-discovery-topics.rst

```diff
@@ -158,3 +152,30 @@ By default, the Docker software does not perform log rotation for the "local/JSO
         "max-file": "3"
       }
     }
+
+==============================
+HTTP/HTTPS Proxy configuration
+==============================
+
+Docker daemon
+-------------
+Docker publishes `detailed instructions <https://docs.docker.com/engine/daemon/proxy/>`_ on how to configure the docker daemon to use a proxy server.
+
+User environment
+----------------
+Ensure that the user environment is configured. This is by ensuring the following environment variables have the correct values.
+
+- HTTP_PROXY
+- http_proxy
+- HTTPS_PROXY
+- https_proxy
+- NO_PROXY
+- no_proxy
+
+Proxy software
+--------------
+The proxy software needs to allow access to three services.
```

Comment:
You may want "...to three services:" instead of "...to three services."

Reply:
Thanks! Updated.